pkg/apis/types.go

```diff
@@ -85,6 +88,19 @@ type RouteConfig struct {
 	Scope uint8 `json:"scope,omitempty"`
 }
 
+// NeighborConfig represents a neighbor (ARP/NDP) entry.
+type NeighborConfig struct {
+	// Destination is the target IP address.
+	Destination string `json:"destination,omitempty"`
+	// HardwareAddr is the MAC address of the neighbor.
+	HardwareAddr string `json:"hardwareAddr,omitempty"`
```

Comment:
we have to update pkg/apis/validation.go to check that both the destination and the hardwareAddress is set

Reply:
Done. Thanks